### DISCUSSION TITLE
Shader Model 6.6, Dynamic Resources

### DISCUSSION THREAD

**Discussion by RobertMtx** [DiligentGraphics/DiligentSamples#106] [Q&A] [answered]
I was wondering if its possible to use the new dynamic resources feature of shader model 6.6 with Diligent, or if that is in the works? I currently know very little about the technical details, but it sounds like a great way to manage resources. I would prefer to start off using this technique, if possible. I couldn't find any of the keywords in the samples or tutorials, so looks like I will be treading off on my own with this.

Here is a link to the details: [Dynamic Resources](https://microsoft.github.io/DirectX-Specs/d3d/HLSL_SM_6_6_DynamicResources.html)

**Comment by TheMostDiligent** [comment, marked as answer]
No, this is not supported. You can use bindless resources though, which are in fact very close to that.
Take a look at [Tutorial 16](https://github.com/DiligentGraphics/DiligentSamples/tree/master/Tutorials/Tutorial16_BindlessResources).
